src/views/projects/SourceBrowser/Changeset.svelte

```diff
@@ -78,14 +106,26 @@
   {/each}
   {#each diff.deleted as file}
     <FileDiff
+      isSelected={file.path == selectedFilename}
+      selectedHunkLine={file.path == selectedFilename
+        ? selectedHunkLine
+        : undefined}
       {projectId}
       {baseUrl}
       {file}
       {revision}
       headerBadgeCaption="deleted" />
   {/each}
   {#each diff.modified as file}
-    <FileDiff {projectId} {baseUrl} {file} {revision} />
+    <FileDiff
+      isSelected={file.path == selectedFilename}
+      selectedHunkLine={file.path == selectedFilename
+        ? selectedHunkLine
+        : undefined}
```

Comment:
is it possible to reference own property? in that case we can re-write:

```
 selectedHunkLine={isSelected ? selectedHunkLine : undefined}
```

Reply:
You can always define a variable `isSelected = file.path == selectedFilename`.